src/pages/logoutPage/Logout.jsx

```diff
@@ -9,22 +8,16 @@ const Logout = () => {
 
      <div className="main">
         <img className="bg-img" src="https://images.unsplash.com/photo-1457530378978-8bac673b8062?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80" alt=""/>
-     {/* </div> */}
 
-        {/* <div className="para"> */}
-
-        <div className="signup-page">
+        <div className="login-page">
             <h2 className="signup-title">Log Out</h2>
-            {/* <input type="email" className="signup-input" placeholder="Email" required/>
-            <input type="password" className="signup-input" placeholder="Password" required/>
-            <div className="checkbox-signup">
-                <input type="checkbox" className="signup" />
-                <label className="signup-label">I agree to the Terms of Service and Privacy Policy</label>
-            </div> */}
             <span className="signup-span">You are Logged Out </span>
-
+            <Link to ="/login">
             <button href="#" className="btn-login">Login</button>
+            </Link>
+            <Link to ="/signup">
             <button href="#" className="btn-login outline">Signup</button>
```

Comment:
Suggestion: if you are using Link then I think `href` attribute has any use here